### ISSUE TITLE
How to decide whether a meaning that is not directly attested may be presupposed

### ISSUE THREAD

**Issue by CasFre** [lingdb/CoBL#114]
Especially for those working on poorly attested languages the following example might be worth discussing:

According to Adams, DictTochB, the Tocharian lexeme anāṣṣäṃ (http://www.cobl.info/lexeme/23282/) is only attested in contexts where it is opposed to a verb meaning 'exhale' and thus anāṣṣäṃ should mean 'inhale'.
However, it does belong to a cognate class with the basic meaning 'breathe', and there is a causative ānäskem derived from it which means 'to make breathe, resuscitate'.

Now the question is: How should branch specialists weight those contradicting arguments in a principled way? Is this case still in line with the description in paragraph two of the guidelines for partially attested languages (https://github.com/lingdb/CoBL/wiki/00--Guidelines-for-Partially-Attested-Languages) and similar enough to the o-ni-ti-ja-pi case? - The difference I see is that the context of the direct attestions here suggests a slightly different (resp. narrower) meaning, while o-ni-ti-ja-pi still means 'decorated with birds', not e.g. 'decorated with eagles' or 'decorated with winged animals'.

**Comment by cormacanderson** [comment]
Perhaps we have to determine this on a case by case basis and leave ultimate responsibility to the branch specialists, but it would be good to discuss this and similar issues. Probably particularly relevant to @MatildeSerangeli @tonyakdewey @m-scarborough . In this specific case, I definitely think we should take it, because the meaning specification requires "inhale" if there is no generic term but rather separate ones for "inhale" and "exhale". @CasFre was there not another case in which there was only an attested word for "exhale"?

**Comment by CasFre** [comment]
@cormacanderson: Yes, this was the Gothic cognate uz-anan from the same cognate set: /lexeme/19933/.
I thought, however, that this might in the end match with the rules set up for partially attested languages, because

1. Gothic uz- is cognate with Germ. aus, Engl. out and thus the compound uz-anan shows the meaning that we would expect from a compounded verb derived from a simple verb meaning 'to breathe' and the prefix 'out'.
Gothic uz- is cognate with Germ. aus, Engl. out and thus the compound uz-anan shows the meaning that we would expect from a compounded verb derived from a simple verb meaning 'to breathe' and the prefix 'out'.

2. It is not harmful that the verb is only attested in the 3. Sg. Pret. uz-on because this is a regular preterite form, cf. again partially attested language policy.

However, discussion of this example is also welcome. Especially, if there are divergent opinions on how to treat this case.

**Comment by tonyakdewey** [comment]
@CasFre, yes, I agree completely with what you've laid out.
